tests/acceptance/sustain-test.js

```diff
@@ -16,7 +17,10 @@ test('visiting /tests/sustain', function(assert) {
 
     assert.ok(id2, 'we found two IDs');
     assert.equal(children.length, 2, 'we have two tagless children');
-    assert.equal(find('.has-model-foo').text(), '123');
+
+    if (hasEmberVersion(2, 0)) {
```

Comment:
Yes, this invocation checks (in a poor mans way) for `>=2.0` ([see really silly simple code here](https://github.com/switchfly/ember-test-helpers/blob/master/lib/ember-test-helpers/has-ember-version.js))